nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -1328,9 +1328,9 @@
                 var tooltip = nf.Common.formatPropertyTooltip(propertyDescriptor, propertyHistory);
 
                 if (nf.Common.isDefinedAndNotNull(tooltip)) {
-                    infoIcon.qtip($.extend({
-                        content: tooltip,
-                    }, nf.Common.config.tooltipConfig));
+                    infoIcon.qtip($.extend(nf.Common.config.tooltipConfig, {
```

Comment:
Ok, nice catch @mcgilman... I have updated all the modified $.extend to create a new object `{}`, extend it with any nf.Common.config.tooltip*, and then extend that with any additional objects to achieve the desired object.